src/lib/isAfter.js

```diff
@@ -1,8 +1,13 @@
 import assertString from './util/assertString';
 import toDate from './toDate';
 
-export default function isAfter(str, date = String(new Date())) {
+export default function isAfter(str, options) {
   assertString(str);
+
+  // accessing 'arguments' for backwards compatibility: isAfter(str [, date])
+  // eslint-disable-next-line prefer-rest-params
+  const date = (typeof options === 'object' ? options.date : arguments[1]) || Date().toString();
```

Comment:
suggestion: In my [PR](https://github.com/validatorjs/validator.js/pull/2088/files#diff-5c43822fb35491823a21d137af899920cddf0533765002ab04e0eb9f5aae7050R8) you can see, that I named this property `comparisonDate`. I find it useful to have a concrete name for an API like this :).

Reply:
Yes, thanks for that suggestion!